### PR TITLE
Update README.md based on feedback of other application integration 

### PR DESCRIPTION
**Pull request by Hedda**
@dmulcahey  Update README.md based on some general feedback from @pipiche38 regarding lack of documentation about integration into other projects/applications that are not ZHA in Home Assistant, for more info see: https://github.com/zigpy/zigpy/issues/452

> One more comment, I wanted to use the Quirks library but this looks ZHA specific, isn't ?

I think this is also related to issue https://github.com/zigpy/zha-device-handlers/issues/327 which is a suggestion to rename the zha-device-handlers library and repository to "zigpy-device-handlers" for clarity and consistency as it is not dependant on ZHA.

Again, the main point with both that and this is just to try to clarify that the zha-quirks library can be used as a standalone library without ZHA in HA as there is no dependence on Home Assistant.

### CODE REVIEW

**Comment by Hedda** [comment]
Please also see the somewhat related https://github.com/zigpy/zha-device-handlers/pull/468

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ ZHA device handlers bridge the functionality gap created when manufacturers devi
...
 
+Note! The ZHA Device Handlers library is a standalone library and not dependent [Home Assistant](https://www.home-assistant.io), however a reference integration implementation of the ZHA Device Handlers library is available in [Home Assistant](https://www.home-assistant.io/) (a Python based open source home automation software) as part of its [ZHA integration component](https://www.home-assistant.io/integrations/zha/). The source code for that reference integration implementation is avalable in Home Assistant's "core" code under the "home-assistant/core/tree/dev/homeassistant/components/zha" directory.
```
Comment (dmulcahey) [inline README.md:11]: This isn't really true. there is no reference implementation in ZHA. All ZHA does is import the library... I think this is misleading. @Adminiuga thoughts?

Comment (pipiche38) [reply README.md:11]: I think that you should refered to 'The Device Handlers library' and not the 'ZHA Device handlers'

Comment (Adminiuga) [reply README.md:11]: Reference implementation using zha device handlers library?

Comment (Hedda) [reply README.md:11]: By "reference implementation" I no not mean to imply that there is any kind of dependency on Home Assistant. I only meant that developers can look at Home Assistant's ZHA code as a reference in order to see how zigpy and zha-quirks can be used.

That terminology wording is used as a standard for these types of scenarios, see  https://en.wikipedia.org/wiki/Reference_implementation

"In the software development process, a reference implementation (or, less frequently, sample implementation or model implementation) is the standard from which all other implementations and corresponding customizations are derived. A reference implementation often accompanies a technical standard, and demonstrates what should be considered the "correct" behaviour of any other implementation of it."

Comment (dmulcahey) [reply README.md:11]: I know what a reference implementation is... this is not one. it's a dependency pure and simple. Again, HA only depends on and imports this library. That's it.

**Comment by dmulcahey** [comment]
Please rebase to pick up the CI change. We swapped from travis to GH Actions. Sorry for the inconvenience.

File: README.md (hunk 1 above)
Comment (Hedda) [reply README.md]: Removed the phrasing "reference implementation".

**Comment by Hedda** [comment]
@dmulcahey What do you think about resolving this pull request merge conflict with https://github.com/zigpy/zha-device-handlers/issues/577 so there would be no need to rebase?

**Comment by Hedda** [comment]
PR replaced with https://github.com/zigpy/zha-device-handlers/pull/578

**Pull request closed without merging**